pkg/health/client/client.go

```diff
@@ -216,3 +216,24 @@ func FormatHealthStatusResponse(w io.Writer, sr *models.HealthStatusResponse, pr
 		fmt.Fprintf(w, "  ...")
 	}
 }
+
+// GetAndFormatHealthStatus fetches the health status from the cilium-health
+// daemon via the default channel and formats its output as a string to the
+// writer.
+//
+// 'succinct', 'verbose' and 'maxLines' are handled the same as in
+// FormatHealthStatusResponse().
```

Comment:
I'd prefer explicit documentation instead of a reference to another function in case `FormatHealthStatusResponse` changes how it behaves.

Reply:
This function passes the arguments unmodified to `FormatHealthStatusResponse()`, if this ever changes then it's the responsibility of the developer and reviewers to ensure the docs are kept up to date. My concern if I duplicate the documentation here then a subsequent update of `FormatHealthStatusResponse()` will update *its* documentation but forget to update this documentation.

Reply:
OK, fair enough. Approved :)